interfaces/acados_template/acados_template/acados_ocp.py

```diff
@@ -1013,17 +1013,17 @@ def generate_external_functions(self, context: Optional[GenerateContext] = None)
         check_casadi_version()
         if self.model.dyn_ext_fun_type == 'casadi':
             if self.solver_options.integrator_type == 'ERK':
-                generate_c_code_explicit_ode(context, model)
+                generate_c_code_explicit_ode(context, model, model_dir)
```

Comment:
Same for the Matlab functions.

Reply:
but this way `model_dir` is defined only once and not five times.

Reply:
one could move it to context though

Reply:
I think we cant, because for multiphase the models vary but there is only one context.
I would prefer setting it up in place still but not too strongly.